src/vs/workbench/browser/parts/editor/editorDropTarget.ts

```diff
@@ -359,7 +359,7 @@ class DropOverlay extends Themable {
 
 			// {{SQL CARBON EDIT}}
 			const editor = this.editorService.activeTextEditorControl as ICodeEditor;
-			if (untitledOrFileResources[0].resource.scheme === 'Column' || untitledOrFileResources[0].resource.scheme === 'Table') {
+			if (untitledOrFileResources[0].resource.scheme === 'Column' || untitledOrFileResources[0].resource.scheme === 'Table' || untitledOrFileResources[0].resource.scheme === 'View' || untitledOrFileResources[0].resource.scheme === 'Folder') {
```

Comment:
Looks like you're checking this condition multiple times, perhaps use a helper function for this

Reply:
This could be 
```
if (this. checkNodeTypeSingle(untitledOrFileResources[0].resource.scheme) || untitledOrFileResources[0].resource.scheme === 'Folder'
```